source/elements/oneDAL/dalapi/generator.py

```diff
@@ -39,7 +39,6 @@ def add_property_member(self, declaration: str, parent_fully_qualified_name: str
         assert declaration
         assert parent_fully_qualified_name
         fake_parent_namespace = '_'.join(parent_fully_qualified_name.split('::'))
```

Comment:
nip: unused variable

Reply:
Should be fixed in the oneDAL master and than synchronized again